force-app/main/default/lwc/applicationSettings/applicationSettings.js

```diff
@@ -0,0 +1,88 @@
+import { LightningElement, wire, api, track } from "lwc";
+import { refreshApex } from "@salesforce/apex";
+
+import getApplicationSettingsVModel from "@salesforce/apex/ApplicationSettingsController.getApplicationSettingsVModel";
+
+// Label imports
+import stgApplicationSettingsTitle from "@salesforce/label/c.stgApplicationSettingsTitle";
+import stgApplicationWindowValidation from "@salesforce/label/c.stgApplicationWindowValidation";
+import stgApplicationWindowValidationDescription from "@salesforce/label/c.stgApplicationWindowValidationDescription";
+
+export default class ApplicationSettings extends LightningElement {
+    isEditMode = false;
+    affordancesDisabledToggle = false;
+
+    @track applicationSettingsVModel;
+    @track _wiredApplicationSettings;
+
+    labelReference = {
+        applicationPageTitle: stgApplicationSettingsTitle,
+        applicationWindowValidationTitle: stgApplicationWindowValidation,
+        applicationWindowValidationDescription: stgApplicationWindowValidationDescription,
+    };
+
+    inputAttributeReference = {
+        enableApplicationWindowValidationId: "enableApplicationWindowValidation",
+    };
+
+    @wire(getApplicationSettingsVModel)
+    wiredApplicationSettings(response) {
+        this._wiredApplicationSettings = response;
+        if (response.data) {
+            this.applicationSettingsVModel = response.data;
+        }
+        if (response.error) {
+            //console.error(response.error);
```

Comment:
I see other *settings component do not display/log an error either. Wondering if errors should be handled in a future WI for EDA Cloud Settings?

Applications settings when data model retrieval generates an exception which results in no data display. Clicking on [Edit] and following through generates an error in console..

https://user-images.githubusercontent.com/28743286/127585482-9caf3d75-05f5-498a-baf8-985d4a444647.mp4

Reply:
We are handling errors in this major release and the next one. The handling got descoped from the prior one.